### ISSUE TITLE
Why does a large gap occur between where2act reported numbers? 

### ISSUE THREAD

**Issue by akira-l** [galaxy-qazzz/AdaAffordCode#7]
![image](https://user-images.githubusercontent.com/9478838/210533311-c70adcaa-9a54-46cf-a159-a8a221416ed7.png)
Above results are from where2act paper 
![image](https://user-images.githubusercontent.com/9478838/210533494-04c637ef-fd20-4572-88df-8b0f55fe4ff0.png)
and these are from adaafford paper. 

There are significant differences between where2act results. Are the results reported in your paper implemented by yourselves? What makes such a difference? 
Moreover, results from the test set even outperformed the training set. Those appear to be wired. Can you explain those?

**Comment by wangyian-me** [comment]
The network in our Where2Act baseline is the same with Where2Act paper.
However, our training dataset and testing dataset are different from Where2Act, which make it unfair if we directly compare to the score in Where2Act paper. Thus, we finetuned Where2Act baseline in our training dataset and evaluated it in our testing dataset. 
In a word, although we were using the same network parameters, our training dataset and testing dataset were completely different, which made the gap in the scores of two papers.

**Comment by wangyian-me** [comment]
The answer of the second problem is quite similar. 
I think it's not so meaningful to compare the score in two different datasets. In fact, in our test categories, there are shapes that is easier to predict (bucket, washing machine), which I think is the main reason for why we achieved higher scores in test category.

**Comment by wangyian-me** [comment]
We focus on the relative scores within one dataset to demonstrate the effectiveness of our design.
Sorry for making you confused.

**Comment by akira-l** [comment]
So you mean the results correspond to generated data samples every time and may fluctuate according to each collection?

**Comment by wangyian-me** [comment]
Yeah

**Comment by wangyian-me** [comment]
This might change if we collect a lot more test data. But it simply requires too much of it, since in our setting we have to collect a batch of data for each shape.

**Comment by akira-l** [comment]
I get it. 
Thanks.
